libfwupdplugin/fu-drm-device.c

```diff
@@ -208,6 +208,7 @@ fu_drm_device_probe(FuDevice *device, GError **error)
 			fu_device_set_name(device, fu_edid_get_eisa_id(edid));
 		if (fu_edid_get_serial_number(edid) != NULL)
 			fu_device_set_serial(device, fu_edid_get_serial_number(edid));
+		fu_device_build_vendor_id(device, "EDID", fu_edid_get_pnp_id(edid));
```

Comment:
@superm1 the other idea would be PNP as the vendor ID base -- i.e. I'm not sure if this should be `where it came from` or `the standards body that assigned it`

Reply:
I know, but which one is _better_? :)

Reply:
I guess I didn't realize they'll be different. I suppose standards body has the argument of rebranded stuff doesn't work?